package.json

```diff
@@ -7,7 +7,8 @@
     "start": "next start",
     "type-check": "tsc",
     "prepare": "husky install",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "test": "jest --watch"
```

Comment:
i think we want `test` as a convention that could be run in ci, so probably dont want it to be in watch mode here.
maybe add a separate `test:watch`

Reply:
Good call.